xml/hardening.xml

```diff
@@ -3445,6 +3445,311 @@ Account Expires: Never</screen>
+  <sect1 xml:id="sec-sec-prot-restrict-root">
+   <title>Restricting &rootuser; logins</title>
+
+   <para>
+    By default the &rootuser; user is assigned a password
+    and can login on any local terminal, in a graphical session or via SSH.
+    For security sensitive systems &rootuser; logins should be
+    restricted to as few methods as possible. Shared usage of the root account
+    should be avoided, instead individual administrators should use tools like
+    <literal>su(1)</literal> or <literal>sudo(8)</literal> to elevate privileges.
+    This allows to associate &rootuser; logins with particular users.
+    This also adds another layer of security, because not only the root password
+    but also the password of an administrator's account needs to be compromised
+    to gain full root access. This section explains how to limit direct root
+    logins on the different levels of the system.
+   </para>
+   <sect2 xml:id="sec-sec-prot-restrict-root-tty">
+    <title>Restricting Local Text Console Logins</title>
+
+    <para>
+     TTY devices provide text mode system access via the console. For desktop
+     systems these are accessed via the local keyboard or in case of server
+     systems via input devices connected to a KVM switch or a remote
+     management card (ILO, DRAC, etc).
+     By default, Linux offers 6 different consoles, that can be switched to
+     via the key-combinations
+     <keycombo><keycap function="alt"/><keycap>F1</keycap></keycombo> to
+     <keycombo><keycap function="alt"/><keycap>F6</keycap></keycombo>, when
+     running in text mode, or
+     <keycombo><keycap function="control"/><keycap function="alt"/><keycap>F1</keycap></keycombo> to
+     <keycombo><keycap function="control"/><keycap function="alt"/><keycap>F6</keycap></keycombo>
+      when running in a graphical session. The
+      associated terminal devices are named <literal>tty1</literal> .. <literal>tty6</literal> accordingly.
+    </para>
+
+    <para>
+     The following steps restrict root access to the first TTY. Even this
+     access method is only meant for emergency access to the system and should
+     never be used for everyday system administration tasks.
+    </para>
+
+    <itemizedlist>
+     <listitem>
+     <para>
+      Ensure that the PAM stack configuration file <filename>/etc/pam.d/login</filename>
+      contains the <literal>pam_securetty</literal> module in the <literal>auth</literal> block:
+     </para>
+     <screen>auth     requisite      pam_nologin.so
+auth     [user_unknown=ignore success=ok ignore=ignore auth_err=die default=bad] pam_securetty.so noconsole
+auth     include        common-auth</screen>
+     </listitem>
+     <listitem>
+      <para>
+       This will include the <literal>pam_securetty</literal> PAM module during the
+       authentication process on local consoles to allow root only to login on
+       tty devices that are listed in the file <filename>/etc/securetty</filename>.
```

Comment:
This is an explanation, not an action. As such, I think this belongs to the first step. I'd suggest to move that as a para into the first step (after the `<screen>` as an additional `<para>`).